falkordb-cluster/cluster-entrypoint.sh

```
@@ -130,6 +130,27 @@ wait_until_node_host_resolves() {
   done
 }
 
+
+reconnect_to_master(){
+  res=$(redis-cli $AUTH_CONNECTION_STRING $TLS_CONNECTION_STRING info replication)
+  echo "THE RESPONSE IS :: "$res" "
```

Shellcheck (0.10.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)